# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/d3d11/CompositorD3D11.cpp
# Commit: 72d4bf5b093e
# Full Hash: 72d4bf5b093e9b00eed78bf4e51850d56df5ae6c
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-05 16:09:03
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add RenderTextureHostSWGL support to D3D11 RenderTextureHosts. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95799
# ==============================================================================

diff -r 99b2e5c55a25 -r 72d4bf5b093e gfx/layers/d3d11/CompositorD3D11.cpp
--- a/gfx/layers/d3d11/CompositorD3D11.cpp	Thu Nov 05 09:14:52 2020 +0000
+++ b/gfx/layers/d3d11/CompositorD3D11.cpp	Thu Nov 05 09:15:05 2020 +0000
@@ -1718,6 +1718,13 @@
   return true;
 }
 
+SyncObjectHost* CompositorD3D11::GetSyncObject() {
+  if (mAttachments) {
+    return mAttachments->mSyncObject;
+  }
+  return nullptr;
+}
+
 void CompositorD3D11::HandleError(HRESULT hr, Severity aSeverity) {
   if (SUCCEEDED(hr)) {
     return;